<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Adminlogin</class>
 <widget class="QWidget" name="Adminlogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>210</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>232</width>
     <height>43</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Admin ID:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="adminIDEdit">
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>251</width>
     <height>43</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Admin Key:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="adminkeyEdit">
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
